modules/manage/pages/manage-security/manage-users-and-roles.adoc

```diff
@@ -131,28 +115,425 @@ on [.ui]*travel-sample*, the view is as follows:
+For example, add an externally authenticated user, with username `externalUserIndividual`, and the `ClusterAdmin` role:
+
+image::manage-security/addNewIndividualLDAPuser.png[,420,align=middle]
+
+Then, left-click on the *Add User* button.
+
+image::manage-security/addUserTab2.png[,120,align=middle]
+
+The *Users & Groups* panel now appears as follows:
+
+image::manage-security/externalUserIndividual.png[,720,align=middle]
+
+The new external user is listed, with their *auth domain* specified as `External`.
+
+[#add-an-externally-authenticated-user-by-group]
+==== Add an Externally Authenticated User with Group-Mapping
+
+If the externally authenticated user _is_ to be associated with an LDAP-group mapping, do not assign roles to the user individually.
+Instead, toggle the right-hand panel, by left-clicking on the *Groups* tab:
+
+image::manage-security/groupsTab.png[,140,align=middle]
+
+This displays the Couchbase-Server user-groups that have been mapped to LDAP groups:
+
+image::manage-security/groupsPanel.png[,170,align=middle]
+
+A single user-group, called `Admins`, is thus shown to have been created.
+For details of the mapping procedure, see xref:manage:manage-security/configure-ldap.adoc[Configure LDAP].
+
+To add an externally authenticated user with this group-mapping, specify the user's username, and check the checkbox for the displayed group.
+Then, left-click on *Add User*, to save.:
+
+image::manage-security/addExternalUserGroup.png[,420,align=middle]
+
+The *Users & Groups* screen now appears as follows:
+
+image::manage-security/usersAndGroupsTwoUsers.png[,800,align=middle]
+
+The new user is duly shown to have the `Full Admin` role associated with the the `Admins` group.
+
+=== Add and Edit Groups
+
+The step-by-step procedures for adding and editing groups are provided in xref:manage:manage-security/configure-ldap.adoc[Configure LDAP].
 
 === Role-Based Console Appearance
 
 Role-assignment determines which features of Couchbase Web Console are
 available to the administrator.
 Non-available features are not displayed: therefore, the console's appearance
-changes, based on which roles have been assigned to the current user.
+changes, based on which roles have been assigned the current user.
+
+[#manage-users-with-the-cli]
+== Manage Users with the CLI
+
+Users can be managed with the xref:cli:cli-user-manage.adoc[user-manage] command.
+This allows the creation and deletion of users and groups, the assignment of roles, and the listing of current status.
+
+[#get-user-information-with-the-cli]
+=== Get User Information with the CLI
+
+To list the cluster's current users, enter the following.
+Note that the command is piped to the `jq` program, to optimize output-readability.
+
+----
+/opt/couchbase/bin/couchbase-cli user-manage --cluster http://10.143.192.101 \
+--username Administrator \
+--password password \
+--list | jq
```

Comment:
not sure we should use jq in documentation examples. Use it only if it is described somewhere what this is and why it is used. I suggest we remove it.
```suggestion
--list
```

Reply:
I've actually used it in quite a few places in the docs generally, in that it's the only way to get some of the output readable. But anyway, I've changed this reference here to do what I normally do, which is link to the external page describing it.